.github/workflows/msbuild.yml

```diff
@@ -47,6 +47,8 @@ jobs:
         repository: boogie-org/boogie
         path: boogie
         ref: v${{ steps.regex-match.outputs.group1 }}
+    - name: Build Java runtime
```

Comment:
YES, I would love that. Worth a time-boxed attempt to get that working if it would mean avoiding adding this to basically every build job, no to mention simplifying the contributor build process.

Note that I'm implicitly dropping the lit behavior in some sub-directories that marks them unsupported if certain language runtimes aren't available. I'm intentionally simplifying things to say that those are hard dependencies and the tests are always run. That makes it reasonable for the `DafnyRuntime` project to have a hard dependency on `javac` in my opinion.